Description: Add support for kFreeBSD/GNU arch
Bug: http://bugs.debian.org/498376
Author: Petr Salinger <Petr.Salinger@seznam.cz>
Last-Update: 2010-07-28
--- a/src/native/unix/support/apsupport.m4
+++ b/src/native/unix/support/apsupport.m4
@@ -73,6 +73,11 @@
     LDFLAGS="$LDFLAGS -ldl"
     supported_os="aix5"
     ;;
+  kfreebsd*-gnu)
+    CFLAGS="$CFLAGS -DOS_BSD -DDSO_DLFCN -pthread"
+    supported_os="kfreebsd-gnu"
+    LIBS="$LIBS -ldl -lpthread"
+    ;;
   *)
     AC_MSG_RESULT([failed])
     AC_MSG_ERROR([Unsupported operating system "$host_os"])
