Index: commons-daemon-1.0.15/src/native/unix/support/apsupport.m4
===================================================================
--- commons-daemon-1.0.15.orig/src/native/unix/support/apsupport.m4     2014-06-19 05:40:23.000000000 +0000
+++ commons-daemon-1.0.15/src/native/unix/support/apsupport.m4  2014-06-19 05:46:23.202817468 +0000
@@ -113,7 +113,7 @@
     LDCMD="/opt/C/bin/cc"
     HOST_CPU=osd
     ;;
-  mips)
+  mips | mipsn32 | mips64)
     CFLAGS="$CFLAGS -DCPU=\\\"mips\\\""
     supported_os="mips"
     HOST_CPU=mips
@@ -142,7 +142,7 @@
     fi
     CFLAGS="$CFLAGS -DCPU=\\\"$HOST_CPU\\\" -DSO_EXT=\\\"sl\\\""
     ;;
-  mipsel)
+  mipsel | mipsn32el | mips64el)
     CFLAGS="$CFLAGS -DCPU=\\\"mipsel\\\""
     supported_os="mipsel"
     HOST_CPU=mipsel
